### DISCUSSION TITLE
Wificonfig 7 and configuration acess point.

### DISCUSSION THREAD

**Discussion by NsinghP** [arendst/Tasmota#12735] [Q&A] [answered]
Hello, 

It was my understanding that by using wificonfig7 I can restrict my device to offer Tasmota configuration Access Point upon reset only. On my 9.5.0 custom build devices I am using "#define WIFI_CONFIG_TOOL WIFI_MANAGER_RESET_ONLY" line. Wificonfig command shows: {"WifiConfig":{"7":"ManagerRst"}}. However, when I turn the home wifi access point off, within few seconds my tasmota device offers Tasmota Configuration Access Point.  

What am I missing? I was expecting that when my Home wifi router is off  my devices will not offer configuration access point if I set wificonfig to 7 and will simply wait for home router to turn back on. 

Setoption 56 is off and Setoption 57 is on.

Regards,

Navi

**Comment by ascillato** [comment, marked as answer]
Sorry, but that is not what wificonfig 7 does.

The docs say:

7 = set Wi-Fi Manager (web server at 192.168.4.1) as the current configuration tool restricted to reset settings only.
This setting is recommended for devices without an external control/reset button.

In other words means that the AP menu is different and will only allows a full reset. Not extra configurations.

If you want that your device never pops up an AP, after its Wi-Fi is configured, you must use the default: `wificonfig 4` 
In wificonfig 4: Tasmota will pop up an AP but ONLY if you reset all configurations (only if the SSID fields are empty in flash).
In case the configured Wi-Fi network can not be reached, Tasmota will try SSID2 (only if SSID2 is configured) and if it can't it will continue searching until any configured SSID will show up again.

**Reply by NsinghP** [reply]
Thanks, I stand corrected. I will now use default value of 4 for wificonfig.
